gplugins/utils/get_component_with_net_layers.py

```diff
@@ -0,0 +1,105 @@
+import copy
+
+import gdsfactory as gf
+from gdsfactory.typings import Component, LayerStack
+
+
+def remove_empty_layerstack_layers(
```

Comment:
double negation is confusin

How about renaming it to `get_component_layer_stack`? in the end you are returning a layerStack for a particular component

Reply:
addressed